src/pymodaq_plugins_keithley/resources/config_template.toml

```diff
@@ -24,28 +24,4 @@ module_name = ""
 
 [Keithley.27XX.INSTRUMENT01.MODULE02.CHANNELS]
 
-[Keithley.27XX.INSTRUMENT01.MODULE01.CHANNELS.101]
-# mode = "temp"
-# transducer = "tc"
-# type = "K"
-# ref_junc = "int"
-# resolution = 6
-# nplc = 5
-
-[Keithley.2100]
```

Comment:
don't you use those configuration?

Reply:
No. Configuration of keithley2100 was removed.